.github/workflows/master.yml

```diff
@@ -389,7 +389,7 @@ jobs:
     # in the process of running or have resulted in failures.
     if: github.event_name == 'pull_request'
     runs-on: ubuntu-latest
-    needs: python
+    needs: windows
```

Comment:
The windows job takes the longest.

Reply:
Any job that can takes a while to finish can be used here. The python is not a bad choice, but not the best.

Reply:
See https://github.com/mlflow/mlflow/pull/10406 how the project job works.